stub-environment/environments/Test-Laptop.json

```diff
@@ -4,7 +4,8 @@
     "bcpc": {
       "domain_name" : "bcpc.example.com",
       "management": {
-        "vip" : "10.0.100.5"
+        "vip" : "10.0.100.5",
+        "viphost": "test-laptop.bcpc.example.com"
```

Comment:
Why not specify only the hostname since the domain is already provided in the environment?